### DISCUSSION TITLE
Поддержка протокола GQUIC Q046

### DISCUSSION THREAD

**Discussion by terloid** [bol-van/zapret#908] [Ideas]
Приложение Youtube (версия 5.13.301) в телевизорах на Google TV 11 использует протокол GQUIC Q046. Прошу добавить распознавание этого протокола. Сейчас использую стратегию --filter-udp=443 --dpi-desync-any-protocol --dpi-desync=fake --dpi-desync-repeats=2 --dpi-desync-fake-unknown-udp="quic_initial_www_google_com.bin", но с ней много лишнего трафика. Виртуалку Google TV 11 для тестирования можно создать в Android Studio.

**Comment by LimpingK** [comment]
Кстати, а с этой стратегией — иногда притормаживает секунд на 10-15 в начале видео?
Или шустро начинает?

**Comment by bol-van** [comment]
там сразу идет шифрованый обмен
даже шарк не может декодировать
хост не извлечь
потому листы не прикрутить
можно сделать распознавание как в вирегуарде
с отдельным фейком

**Reply by terloid** [reply]
> там сразу идет шифрованый обмен даже шарк не может декодировать хост не извлечь потому листы не прикрутить можно сделать распознавание как в вирегуарде с отдельным фейком

первый исходящий пакет в нем не шифруется, хост открыт, вот пример https://drive.google.com/file/d/15-kUwUKWA0WIbm0314eeIVA20FGBys_M/view?usp=sharing
этого недостаточно?

**Reply by bol-van** [reply]
как я понял он сохраняет что-то и в новых сеансах сразу крипто

**Reply by terloid** [reply]
> можно сделать распознавание как в вирегуарде с отдельным фейком

Вариант приемлемый. Владельцы теликов будут благодарны.

**Comment by bol-van** [comment]
ани протокол всегда надо ограничивать коннбайтс или катофф
иначе трэш флуд
